packages/starlight/components/Icons.ts

```diff
@@ -64,6 +64,8 @@ export const BuiltInIcons = {
 		'<path d="M1 1.5a.8.8 0 0 0-.7.9l3.2 19.3c0 .5.5.8 1 .8h15.2c.4 0 .7-.2.8-.6l3.2-19.5a.7.7 0 0 0-.8-.9H1zm13.4 14H9.6l-1.3-7h7.3l-1.2 7z"/>',
 	codePen:
 		'<path d="M23.5 7.5 12.5.2a1 1 0 0 0-1 0L.4 7.5a1 1 0 0 0-.5.8v7.4c0 .3.2.6.5.8l11 7.3c.3.3.7.3 1 0l11-7.3c.3-.2.5-.5.5-.8V8.3a1 1 0 0 0-.5-.8zM13 3l8.1 5.3-3.6 2.5-4.5-3V3zm-2 0v4.8l-4.5 3-3.6-2.5 8-5.3zm-9 7.3L4.7 12l-2.5 1.7v-3.4zM11 21l-8.1-5.3 3.6-2.5 4.5 3V21zm1-6.6L8.4 12 12 9.6l3.6 2.4-3.6 2.4zm1 6.6v-4.8l4.5-3 3.6 2.5-8 5.3zm9-7.3L19.3 12l2.5-1.7v3.4z"/>',
+	farcaster:
+		'<path d="M6.187 3.733h11.627v16.533h-1.707V12.693h-0.017c-0.189 -2.093 -1.948 -3.733 -4.09 -3.733s-3.901 1.64 -4.09 3.733h-0.017v7.573h-1.707z" fill="#855DCD"></path> <path d="m3.093 6.08 0.693 2.347h0.587v9.493c-0.295 0 -0.533 0.239 -0.533 0.533v0.64h-0.107c-0.295 0 -0.533 0.239 -0.533 0.533v0.64h5.973v-0.64c0 -0.295 -0.239 -0.533 -0.533 -0.533h-0.107v-0.64c0 -0.295 -0.239 -0.533 -0.533 -0.533h-0.64V6.08z" fill="#855DCD"></path> <path d="M16.213 17.92c-0.295 0 -0.533 0.239 -0.533 0.533v0.64h-0.107c-0.295 0 -0.533 0.239 -0.533 0.533v0.64h5.973v-0.64c0 -0.295 -0.239 -0.533 -0.533 -0.533h-0.107v-0.64c0 -0.295 -0.239 -0.533 -0.533 -0.533V8.427h0.587L21.12 6.08H16.853v11.84z" fill="#855DCD"></path>',
```

Comment:
```suggestion
		'<path d="M6.187 3.733h11.627v16.533h-1.707v-7.573h-.017a4.107 4.107 0 0 0-8.18 0h-.017v7.573H6.186z"/><path d="m3.093 6.08.693 2.347h.587v9.493a.533.533 0 0 0-.533.533v.64h-.107a.533.533 0 0 0-.533.533v.64h5.973v-.64a.533.533 0 0 0-.533-.533h-.107v-.64A.533.533 0 0 0 8 17.92h-.64V6.08zM16.213 17.92a.533.533 0 0 0-.533.533v.64h-.107a.533.533 0 0 0-.533.533v.64h5.973v-.64a.533.533 0 0 0-.533-.533h-.107v-.64a.533.533 0 0 0-.533-.533V8.427h.587l.693-2.347h-4.267v11.84z"/>',
```

Thanks for the update. The SVG does indeed now properly shows up but would require a few more tiny changes:

- The hardcoded color should be removed.
- We can still optimize the resized SVG a bit using [SVGOMG](https://jakearchibald.github.io/svgomg/)

The suggested change I submitted contains this and renders as follows:

<img width="174" alt="image" src="https://github.com/withastro/starlight/assets/494699/dcce2eed-fa80-4284-8686-ca2efd18fd19">